66.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {number[]} digits
+ * @return {number[]}
+ */
+var plusOne = function(digits) {
+    for(let i = digits.length-1; i >= 0; i--) {
+        if(digits[i] != 9) {
+            digits[i]++;
+            return digits;
```

Comment:
You can also use `break` instead of `return digits` because `break` here leads to `return digits` in Line 18, which ends up in the same result. This is just my preference so you don't necessarily have to change it though.